cmd/skopeo/delete.go

```diff
@@ -18,7 +18,14 @@ func deleteHandler(context *cli.Context) error {
 		return fmt.Errorf("Invalid source name %s: %v", context.Args()[0], err)
 	}
 
-	if err := ref.DeleteImage(contextFromGlobalOptions(context)); err != nil {
+	ctx := contextFromGlobalOptions(context)
+	if context.IsSet("creds") {
+		ctx.DockerAuthConfig, err = getDockerAuth(context.String("creds"))
```

Comment:
If everyone is going to use it like this, shouldn't it be folded into `contextFromGlobalOptions(context, creds-option-name)`?